OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -421,7 +421,11 @@ public interface ITargetablePositions
 		bool AlwaysEnabled { get; }
 	}
 
-	public interface IMoveInfo : ITraitInfoInterface { }
+	public interface IMoveInfo : ITraitInfoInterface
+	{
+		bool GetMoveIntoShroud();
```

Comment:
Imho this would make more sense as `CanMoveIntoShroud`. Any other opinions?

Reply:
Makes sense to me. Renamed.